### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by neophob**
minor cleanup:
- move strict statement on top of classes
- replace usage of console.log with debug
- removed unused imports
- add removeAllBridgedAccessories function to remove all added accessoires 
- bump node-persistence
- minor update of the readme

### CODE REVIEW

File: lib/Accessory.js [hunk 1]
```diff
@@ -11,9 +13,6 @@ var Advertiser = require('./Advertiser').Advertiser;
 var HAPServer = require('./HAPServer').HAPServer;
 var AccessoryInfo = require('./model/AccessoryInfo').AccessoryInfo;
 var IdentifierCache = require('./model/IdentifierCache').IdentifierCache;
-// var RelayServer = require("./util/relayserver").RelayServer;
```
Comment (KhaosT) [inline lib/Accessory.js]: Please don't remove this one, while it's not being publicly distributed, it's being used somewhere.

File: lib/gen/HomeKitTypes.js [hunk 2]
```diff
@@ -640,6 +641,7 @@ Characteristic.CurrentTemperature = function() {
     format: Characteristic.Formats.FLOAT,
     unit: Characteristic.Units.CELSIUS,
     maxValue: 100,
+    //TODO see https://github.com/plavrenko/HAP-NodeJS/commit/53bc5635ed453135d6e48759cd987bae2a97c407
```
Comment (KhaosT) [inline lib/gen/HomeKitTypes.js]: Why do you add this?

File: package.json [hunk 3]
```diff
@@ -13,8 +13,8 @@
     "ed25519": "^0.0.4",
     "fast-srp-hap": "^1.0.0",
     "ip": "^1.1.3",
-    "mdns": "^2.3.2",
-    "node-persist": "^0.0.8"
+    "mdns": "^2.3.3",
+    "node-persist": "^0.0.11"
```
Comment (KhaosT) [inline package.json]: Maybe we should try and see if we can migrate to latest node-persist. It's fine to use 0.0.11 for this PR.

Comment (neophob) [inline package.json]: yeah, I guess it make sense to update node-persist in a separate PR

File: lib/gen/HomeKitTypes.js (hunk 2 above)
Comment (neophob) [inline lib/gen/HomeKitTypes.js]: ok will remove this, was more a todo for myself I forgot to add

File: lib/Accessory.js (hunk 1 above)
Comment (neophob) [inline lib/Accessory.js]: ok, I'll re-add this. maybe control this with a env variable would make it easier to understand for the future

**Comment by neophob** [comment]
@KhaosT updated the two issues you mentioned

**Comment by KhaosT** [comment]
Thank You :)

**Pull request merged**